src/Twig/Extension/PaginationRuntime.php

```diff
@@ -109,20 +109,26 @@ public function filter(
     }
 
     /**
-     * @param array<string, mixed> $query
-     * @param int $page
+     * @param array<string, mixed>      $query
+     * @param int                       $page
+     * @param array<string, mixed>      $options
      * @return array<string, mixed>
      */
-    public function getQueryParams(array $query, int $page): array
+    public function getQueryParams(array $query, int $page, array $options): array
```

Comment:
sorry @steveWinter I was about to release this change, but I realized this new argument would break any custom template not passing it.
We need to fix it with a default value of `[]`